samples/blueprint.yaml

```diff
@@ -3,28 +3,35 @@ tosca_definitions_version: cloudify_dsl_1_2
 imports:
   - http://www.getcloudify.org/spec/cloudify/3.3/types.yaml
   - http://www.getcloudify.org/spec/psutil-plugin/1.0/plugin.yaml
+  - http://www.getcloudify.org/spec/aws-plugin/1.3.1/plugin.yaml
```

Comment:
Well, that was a mistake and it probably should be like this. It doesn't work this way, but when you update it to the version which contains automatic password retrieval from Windows machines it should work.